tests/test_jsondiff.py

```diff
@@ -8,6 +9,15 @@
 from hypothesis import given, settings, strategies
 
 
+logging.basicConfig(
```

Comment:
This seems like debugging code. Did you intend to leave this in here?

Reply:
It's convenient to add some log during test.